.github/workflows/check.yml

```diff
@@ -5,72 +5,66 @@ env:
 
 jobs:
   check:
-    name: Neqo Build and Test
-    runs-on: ubuntu-22.04
-    env:
-      RUSTFLAGS: -C link-arg=-fuse-ld=lld
+    name: Build & test
     strategy:
+      fail-fast: false
       matrix:
-        rust-toolchain: [stable, 1.65.0]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        rust-toolchain: [1.65.0] #, stable] #, beta]
```

Comment:
Any reason that you dropped stable from this?

Reply:
Oops. I'll put it back.